src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts

```diff
@@ -56,11 +63,28 @@ export class HuiLightCardEditor extends LitElement
     return this._config!.icon || "";
   }
 
+  get _hold_action(): ActionConfig {
+    return this._config!.hold_action || { action: "none" };
+  }
+
+  get _double_tap_action(): ActionConfig {
+    return this._config!.double_tap_action || { action: "none" };
+  }
+
   protected render(): TemplateResult {
     if (!this.hass) {
       return html``;
     }
 
+    const actions = [
+      "more-info",
+      "toggle",
+      "navigate",
+      "url",
+      "call-service",
+      "none",
+    ];
```

Comment:
Why haven't we extracted this somewhere? The actions are mainly the same for all cards?

Reply:
I can do that in another PR